ReactiveFeedback/Feedback.swift

```diff
@@ -28,15 +46,16 @@ public struct Feedback<State, Event> {
     ///              `transform` and yielding events that eventually affect
     ///              the state.
     public init<U, Effect: SignalProducerConvertible>(
-        deriving transform: @escaping (Signal<State, Never>) -> Signal<U, Never>,
+        compacting transform: @escaping (SignalProducer<State, Never>) -> SignalProducer<U, Never>,
         effects: @escaping (U) -> Effect
     ) where Effect.Value == Event, Effect.Error == Never {
-        self.events = { scheduler, state in
+        self.events = { state, scheduler, output in
             // NOTE: `observe(on:)` should be applied on the inner producers, so
             //       that cancellation due to state changes would be able to
             //       cancel outstanding events that have already been scheduled.
-            return transform(state)
-                .flatMap(.latest) { effects($0).producer.observe(on: scheduler) }
+            transform(state)
+                .flatMap(.latest) { effects($0).producer.observe(on: scheduler).enqueue(to: output) }
```

Comment:
Hey @andersio, I've added a scheduler parameter and default to `ImmediateScheduler` so we can retain previous behaviour when custom scheduler provided. What do you think?

Reply:
LGTM

Reply:
I would recommend implementing it in `enqueue(to:)` though.